### PR TITLE
issue/1710 – Added Paid Member Subscriptions integration

### PR DESCRIPTION
**Pull request by IovaMihai**
Issue: #1710

### CODE REVIEW

**Comment by michaelbeil** [comment]
Issue https://github.com/AffiliateWP/AffiliateWP/issues/1710

File: includes/class-integrations.php [hunk 1]
```diff
@@ -33,6 +33,7 @@ public function get_integrations() {
 			'wpec'           => 'WP eCommerce',
 			'wp-invoice'     => 'WP-Invoice',
 			'zippycourses'   => 'Zippy Courses',
+            'pms'            => 'Paid Member Subscriptions'
```
Comment (michaelbeil) [inline includes/class-integrations.php]: @IovaMihai Can you align this?

Comment (michaelbeil) [inline includes/class-integrations.php]: Also, it should be alphabetized.

Comment (IovaMihai) [inline includes/class-integrations.php]: Hey @michaelbeil , done! Thanks for letting me know about this.

**Pull request merged**